### PR TITLE
Fix #1979 - Improve UI of call for speakers

### LINKED ISSUE #1979: Improve UI of Call for Speaker
**Actual Behaviour**

The current UI of Call for Speaker has bugs and can be improved.

**Expected Behaviour**

Better UI.

**Steps to reproduce it**

Click Call for Speaker from the menu.

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #1979 

Changes: Make Card Layout the base layout.
Add padding to heading
Make header bold.

Screenshots for the change: 
image

### CODE REVIEW

**Comment by shreyasnbhat** [comment]
@mayank8318 Just a suggestion.You could format the card a bit better.The text inside the card could be neatly arranged.

**Comment by mayank8318** [comment]
@shreyasnbhat What I wanted to do was to show the email - id on a new line in bold text but the entire "Announcement" string is coming from the server I guess, so I couldn't format it.
Am I right or is it hardcoded somewhere?

**Comment by iamareebjamal** [comment]
Increase the padding a little bit please

**Comment by mayank8318** [comment]
Ok 👍

**Comment by mayank8318** [comment]
image

File: android/app/src/main/res/layout/speakers_call_dialog.xml [hunk 1]
```diff
@@ -1,39 +1,55 @@
...
         <TextView
-            android:layout_width="0dp"
-            android:layout_weight="1"
+            android:id="@+id/holder_textview"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="10sp"
-            android:id="@+id/from_date_textview" />
+            android:background="@color/color_primary"
+            android:padding="16dp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/speakers_call_dialog.xml]: Please use dimens value

File: android/app/src/main/res/layout/speakers_call_dialog.xml [hunk 2]
```diff
@@ -1,39 +1,55 @@
...
+            android:textAlignment="center"
+            android:textColor="@color/card_title"
+            android:textSize="20sp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/speakers_call_dialog.xml]: Please use a pre-existing style from compat library

File: android/app/src/main/res/layout/speakers_call_dialog.xml [hunk 3]
```diff
@@ -1,39 +1,55 @@
...
+            android:textStyle="bold"
+            tools:text="Header" />
+
         <TextView
-            android:layout_width="0dp"
-            android:layout_weight="1"
+            android:id="@+id/announcement"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="16dp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/speakers_call_dialog.xml]: Same here

File: android/app/src/main/res/layout/speakers_call_dialog.xml [hunk 4]
```diff
@@ -1,39 +1,55 @@
...
+            tools:text="Announcement" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="10sp"
-            android:id="@+id/to_date_textview" />
+            android:padding="16dp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/speakers_call_dialog.xml]: Same here

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/layout/speakers_call_dialog.xml (hunk 4 above)
Comment (mayank8318) [reply android/app/src/main/res/layout/speakers_call_dialog.xml]: Ok :+1:

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**